CHANGELOG.asciidoc

```diff
@@ -3,6 +3,220 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.16.0]]
+=== Beats version 7.16.0
+https://github.com/elastic/beats/compare/v7.15.2...v7.16.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Skip add_kubernetes_metadata processor when kubernetes metadata are already present {pull}27689[27689]
```

Comment:
@MichaelKatsoulis is this a breaking change?

Reply:
No it is not. It just prevents the kubernetes metadata to get overridden by the processor in case they are already there. In agent this was causing  filebeat to keep logging [errors](https://github.com/elastic/beats/issues/27216)